apps/projects/app/utils/apollo-client.js

```diff
@@ -1,16 +1,34 @@
 import ApolloClient from 'apollo-boost'
+import { useAragonApi } from '../api-react'
+import { GITHUB_TOKEN_REVOKED } from '../store/eventTypes'
+import { STATUS } from '../utils/github'
 
-export const initApolloClient = token =>
-  new ApolloClient({
-    uri: 'https://api.github.com/graphql',
-    request: operation => {
-      if (token) {
-        operation.setContext({
-          headers: {
-            accept: 'application/vnd.github.starfire-preview+json', // needed to create issues
-            authorization: `bearer ${token}`,
-          },
-        })
+export const useApolloClient = () => {
+  const { api } = useAragonApi()
+  const initApolloClient = token => {
+    if (token === null) return null
+    return new ApolloClient({
+      uri: 'https://api.github.com/graphql',
+      request: operation => {
+        if (token) {
+          operation.setContext({
+            headers: {
+              accept: 'application/vnd.github.starfire-preview+json', // needed to create issues
+              authorization: `bearer ${token}`,
+            },
+          })
+        }
+      },
+      onError: error => {
+        if (error.networkError && error.networkError.statusCode === 401) {
+          api.emitTrigger(GITHUB_TOKEN_REVOKED, {
+            status: STATUS.REVOKED,
+            scope: null,
+            token: null,
+          })
+        }
```

Comment:
Let's add an `else` with `console.error(error)`, to keep the functionality we had before and make sure these errors are still visible in the console